### DISCUSSION TITLE
Group focus-visible utility

### DISCUSSION THREAD

**Discussion by agusterodin** [tailwindlabs/tailwindcss#4991] [Ideas]
A group-focus-visible utility would be really nice to have

**Comment by adamwathan** [comment]
This is already included if you use the JIT engine, which will be the default for Tailwind 3.0 in the fall 👍

**Reply by agusterodin** [reply]
Nice! Thanks Adam.

Looks like Create React App will work with JIT in the very near future (definitely by fall) as well. The maintainers have been going crazy the last few weeks in an effort to get an alpha out with Webpack 5 + PostCSS 8. 

I am not using CRA by choice. Changing part of tech stack is a tough sell to management (oftentimes for good reason).

**Reply by dcastil** [reply]
@adamwathan Do you think it would make sense to add that info to the documentation? I was searching for this feature but didn't find anything about it (which brought me here 😄), not even on the JIT page.

On the JIT page it is mentioned [here](https://tailwindcss.com/docs/just-in-time-mode#sibling-selector-variants):

> Similar to the `group-*` variants we’ve supported for years for styling an element based on the parent state, you can use the new `peer-*` variants to style an element based on the state of one of its previous siblings:
